src/test/test-libmount.c

```diff
@@ -28,7 +30,12 @@ static void test_libmount_unescaping_one(
         const char *source, *target;
         _cleanup_free_ char *x = NULL, *cs = NULL, *s = NULL, *ct = NULL, *t = NULL;
 
-        assert_se(mnt_table_next_fs(table, iter, &fs) == 0);
+        r = mnt_table_next_fs(table, iter, &fs);
+        if (r < 0 && may_fail) {
```

Comment:
can you add a comment here maybe, that says "we allow this to fail in some tests, see below", or so?